bundles/org.openhab.voice.voicerss/README.md

```diff
@@ -25,13 +29,135 @@ https://api.voicerss.org/?key=API_KEY&hl=de-de&f=44khz_16bit_mono&src=Hallo%20We
 
 ## Configuration
 
-You must add your API_KEY to your configuration by adding a file "voicerss.cfg" to the services folder, with this entry:
+The following configurations can be edited using the "VoiceRSS Text-to-Speech" settings in the UI:
```

Comment:
Just to be sure, I'm on release 3.1.1 and I don't see any parameter besides version in the VoiceRSS settings. Is this something that has been changed after 3.1.1 ?

Reply:
No, it is there since always.

Reply:
Try to clar your WEB browser cache.

![image](https://user-images.githubusercontent.com/10186704/150650579-5ebdbf23-3daa-4a03-b168-7ee4d48e8432.png)

Reply:
Because I did not look at the right place ! There is actually 3 locations that refer to VoiceRss, the add-ons installation (there is nothing to configure there), the System Services -> Voice where you can configure the actual voice and a few other parameters, and lastly Other Service -> Voice RSS where you can configure the API key.

This shall be made clearer in the doc, especially because voice and API key are not in the same location.